### PR TITLE
Pods 5507

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/identifiers/establishers/individual/address/AddressId.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+case class AddressId(index: Int) extends TypedIdentifier[Address] {
+  override def path: JsPath =
+    EstablishersId(index).path \ AddressId.toString
+}
+
+object AddressId {
+  override lazy val toString: String = "address"
+  def collectionPath(index: Int): JsPath =
+    EstablishersId(index).path \ AddressId.toString
```
Comment (vinnicombe) [inline app/identifiers/establishers/individual/address/AddressId.scala:32]: I saw in another PR comment that collection path is not needed, same here? And for other IDs in this PR

File: app/identifiers/establishers/individual/details/EstablisherDOBId.scala~HEAD [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
```
Comment (vinnicombe) [inline app/identifiers/establishers/individual/details/EstablisherDOBId.scala~HEAD:2]: Why are these ~HEAD files here?

File: app/identifiers/establishers/individual/address/AddressId.scala (hunk 1 above)
Comment (grantrobson) [reply app/identifiers/establishers/individual/address/AddressId.scala]: I have removed and run unit tests. All fine

File: app/identifiers/establishers/individual/details/EstablisherDOBId.scala~HEAD (hunk 2 above)
Comment (grantrobson) [reply app/identifiers/establishers/individual/details/EstablisherDOBId.scala~HEAD]: Removed

**Review by vinnicombe — APPROVED** [review]

**Pull request merged**